catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/view/ModalRegistry.view.js

```diff
@@ -131,6 +125,34 @@ function (ich,Marionette,Backbone,ConfigurationEdit,wreqr,_,$,Utils,Service,moda
                     });
             }
         },
+        urlChanged: function(evt) {
+            var url = evt.target.value;
+            var regExUrl;
+            if (url) {
+                regExUrl = this.extractHostnameFromUrl(url);
+            }
+            var hostname;
+            if (regExUrl) {
+                hostname = regExUrl;
+                if (hostname) {
+                    this.checkName(hostname);
+                } else {
```

Comment:
Ignoring that this else won't get hit as currently written, should something happen with the url other than just validating it?